src/Compilers/Core/Portable/InternalUtilities/NoThrowStream.cs

```diff
@@ -0,0 +1,31 @@
+﻿
+using System;
+using System.IO;
+
+namespace Roslyn.Utilities
+{
+    struct NoThrowStreamDisposer : IDisposable
```

Comment:
Could this derive from Stream and override dispose?

---
In reply to: [96550145](https://github.com/dotnet/roslyn/pull/16574#discussion_r96550145) [](ancestors = 96550145)